package.json

```diff
@@ -43,81 +43,76 @@
+    "download": "5.0.3",
+    "envify": "4.1.0",
+    "eslint": "1.10.3",
     "eslint-config-airbnb": "0.0.8",
-    "eslint-config-algolia": "^3.0.0",
-    "gh-pages": "^1.0.0",
-    "gulp": "^3.9.1",
-    "gulp-cli": "^1.2.1",
-    "gulp-livereload": "^3.8.1",
-    "gulp-sass": "^2.3.2",
-    "handlebars": "^4.0.5",
-    "http-server": "^0.9.0",
-    "jade": "^1.11.0",
-    "jsdoc-parse": "^1.2.7",
-    "jsdoc-to-markdown": "^1.3.6",
-    "marked": "^0.3.6",
-    "metalsmith": "^2.1.0",
-    "metalsmith-changed": "^2.0.0",
-    "metalsmith-discover-partials": "^0.1.0",
-    "metalsmith-headings": "^0.1.0",
+    "eslint-config-algolia": "3.0.0",
+    "gh-pages": "1.0.0",
+    "gulp": "3.9.1",
+    "gulp-cli": "1.3.0",
+    "gulp-livereload": "3.8.1",
+    "gulp-sass": "2.3.2",
+    "handlebars": "4.0.10",
+    "http-server": "0.9.0",
+    "jade": "1.11.0",
+    "jsdoc-parse": "1.2.7",
+    "jsdoc-to-markdown": "1.3.9",
+    "marked": "0.3.6",
+    "metalsmith": "2.3.0",
+    "metalsmith-changed": "2.0.0",
+    "metalsmith-discover-partials": "0.1.0",
+    "metalsmith-headings": "0.1.0",
     "metalsmith-in-place": "superwolff/metalsmith-in-place#7cb06e54142b8843f35178ceb5560946ae356049",
-    "metalsmith-layouts": "^1.6.5",
-    "metalsmith-markdown": "^0.2.1",
-    "metalsmith-metallic": "^1.0.0",
-    "metalsmith-packagejson": "^1.0.1",
-    "metalsmith-sass": "^1.3.0",
+    "metalsmith-layouts": "1.8.1",
+    "metalsmith-markdown": "0.2.1",
+    "metalsmith-metallic": "1.0.0",
+    "metalsmith-packagejson": "1.1.0",
+    "metalsmith-sass": "1.4.0",
     "metalsmith-serve": "0.0.7",
-    "metalsmith-watch": "^1.0.3",
-    "metalsmith-webpack": "^1.0.3",
-    "mkdirp": "^0.5.1",
-    "mversion": "^1.10.1",
-    "onchange": "^3.0.0",
-    "opn-cli": "^3.1.0",
-    "phantomjs-prebuilt": "^2.1.7",
-    "pretty-bytes-cli": "^2.0.0",
-    "prompt": "^1.0.0",
+    "metalsmith-watch": "1.0.3",
+    "metalsmith-webpack": "1.0.3",
+    "mkdirp": "0.5.1",
+    "mversion": "1.10.1",
+    "onchange": "3.2.1",
+    "opn-cli": "3.1.0",
+    "pretty-bytes-cli": "2.0.0",
+    "prompt": "1.0.0",
     "replace-ext": "1.0.0",
-    "rimraf": "^2.5.2",
-    "semver": "^5.1.0",
-    "shelljs": "^0.7.8",
-    "sinon": "^4.4.9",
-    "st": "^1.1.0",
-    "tap-spec": "^4.1.1",
-    "tape": "^4.5.1",
-    "uglify-js": "^2.6.2",
-    "vinyl": "^1.1.1",
-    "watchify": "^3.7.0",
-    "webpack": "^1.13.1",
-    "webpack-stream": "^3.2.0",
-    "zuul": "^3.10.1",
-    "zuul-ngrok": "^4.0.0"
+    "rimraf": "2.6.1",
+    "semver": "5.3.0",
+    "shelljs": "0.7.8",
+    "sinon": "4.4.9",
+    "st": "1.2.0",
+    "tap-spec": "4.1.1",
+    "tape": "4.7.0",
+    "uglify-js": "2.8.29",
+    "vinyl": "1.2.0",
+    "watchify": "3.7.0",
+    "webpack": "1.15.0",
+    "webpack-stream": "3.2.0",
+    "zuul": "^3.11.1",
+    "zuul-ngrok": "4.0.0"
   },
   "dependencies": {
-    "events": "^1.1.0",
-    "lodash": "^4.13.1",
-    "qs": "^6.2.1",
-    "util": "^0.10.3"
```

Comment:
Is it intended to remove this dependencies? Since we are still using it.

https://github.com/algolia/algoliasearch-helper-js/blob/d7985565d2a28e677e7ea56badcefda41a8f5b30/src/DerivedHelper/index.js#L2